<!-- sct-test-summary -->
## :x: Test Summary: FAILED

### :white_check_mark: Precommit: PASSED

| Total | Passed | Failed | Skipped |
|-------|--------|--------|---------|
| 54 | 18 | 0 | 36 |


### :x: Tests: FAILED

| Total | Passed | Failed | Errors | Skipped |
|-------|--------|--------|--------|---------|
| 4605 | 4541 | 3 | 9 | 52 |

### Failed Tests

#### `integration.test_aws_services.test_01_keystore`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_02_keystore_sync`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_03_provision[on_demand]`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_03_provision[spot]`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_03_provision[spot_fleet]`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_04_get_scylla_ami_versions`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_04b_get_scylla_ami_versions_full_tag`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_aws_services.test_05_ec2_client_spot`
**Type:** ERROR
**Message:** `failed on setup with "botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret."`
<details><summary>Traceback</summary>

```
@pytest.fixture(scope="session", autouse=True)
    def fixture_get_real_keys():
>       KeyStore().sync(
            keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519", "scylla_test_id_ed25519.pub"],
            local_path=Path("~/.ssh/").expanduser(),
            permissions=0o0600,
        )

unit_tests/integration/test_aws_services.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm_version_id(key)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:336: in wrapped_f
    return copy(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:475: in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/site-packages/tenacity/__init__.py:376: in 
```
</details>

#### `integration.test_ndbench_thread.test_02_cql_kill`
**Type:** FAILURE
**Message:** `TimeoutError: 1 (of 1) futures unfinished`
<details><summary>Traceback</summary>

```
request = <FixtureRequest for <Function test_02_cql_kill>>
docker_scylla = <sdcm.utils.docker_remote.RemoteDocker object at 0x7b5728869450>
params = SCTConfiguration(perf_simple_query_extra_command=None, force_run_iotune=None, data_volume_disk_throughput=None, multi_...d_regions=['eu-west-1', 'eu-west-2', 'eu-west-3', 'us-west-2', 'us-east-1', 'us-east-2', 'eu-north-1', 'eu-central-1'])

    def test_02_cql_kill(request, docker_scylla, params):
        """
        verifies that kill command on the NdBenchStressThread is working
        """
        loader_set = LocalLoaderSetDummy(params=params)
        cmd = (
            "ndbench cli.clientName=CassJavaDriverGeneric ; numKeys=1000 ; "
            "numReaders=2; numWriters=2 ; cass.writeConsistencyLevel=QUORUM ; "
            "cass.readConsistencyLevel=QUORUM ; readRateLimit=500 ; writeRateLimit=500"
        )
        ndbench_thread = NdBenchStressThread(loader_set, cmd, node_list=[docker_scylla], timeout=500, params=params)
    
        def cleanup_thread():
            ndbench_thread.kill()
    
        request.addfinalizer(cleanup_thread)
        ndbench_thread.run()
        time.sleep(3)
        ndbench_thread.kill()
>       ndbench_thread.get_results()

unit_tests/integration/test_ndbench_thread.py:50: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/stress/base.py:111: in get_results
    for future in concurrent.futures.as_completed(self.results_futures, timeout=timeout):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

fs = {<Future at 0x7b57289e7e50 state=running>}, timeout = 850

    def as_completed(fs, timeout=None):
        """An iterator over the given futures that yields each as it completes.
    
        Args:
            fs: The sequence of Futures (possibly created by different Executors) to
                iterate over.
            timeout: 
```
</details>

#### `integration.test_sync.test_multiple_sync_on_lots_of_files`
**Type:** FAILURE
**Message:** `botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeSecret operation: Secrets Manager can't find the specified secret.`
<details><summary>Traceback</summary>

```
@pytest.mark.integration
    def test_multiple_sync_on_lots_of_files():
        """
        this start multiple client of boto3 in multiple thread,
        opening boto3 session isn't thread safe, so this test is to make sure our lock is working
        """
    
        def is_using_aws_mock():
            try:
                socket.gethostbyname("aws-mock.itself")
                return True
            except socket.gaierror:
                return False
    
        if not is_using_aws_mock():
            key_store = KeyStore()
>           key_store.sync(
                keys=["scylla-qa-ec2", "scylla-test", "scylla_test_id_ed25519"] * 20,
                local_path=Path("~/.ssh/").expanduser(),
                permissions=0o0600,
            )

unit_tests/integration/test_sync.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
sdcm/keystore.py:443: in sync
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
/usr/local/lib/python3.14/concurrent/futures/_base.py:639: in result_iterator
    yield _result_or_cancel(fs.pop())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:311: in _result_or_cancel
    return fut.result(timeout)
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:450: in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/_base.py:395: in __get_result
    raise self._exception
/usr/local/lib/python3.14/concurrent/futures/thread.py:86: in run
    result = ctx.run(self.task)
             ^^^^^^^^^^^^^^^^^^
/usr/local/lib/python3.14/concurrent/futures/thread.py:73: in run
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:443: in <lambda>
    list(executor.map(lambda p: self.get_obj_if_needed(*p), args))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sdcm/keystore.py:419: in get_obj_if_needed
    remote_version = self.get_sm
```
</details>

#### `integration.test_gemini_thread.test_gemini_write_only_with_oracle@docker_heavy`
**Type:** FAILURE
**Message:** `AssertionError: Gemini write-only run failed: {'results': [], 'errors': {}, 'status': 'FAILED'}
assert 'FAILED' == 'PASSED'
  
  - PASSED
  + FAILED`
<details><summary>Traceback</summary>

```
request = <FixtureRequest for <Function test_gemini_write_only_with_oracle>>
docker_scylla = <sdcm.utils.docker_remote.RemoteDocker object at 0x703eb826c050>
docker_scylla_oracle = <sdcm.utils.docker_remote.RemoteDocker object at 0x703eb826c910>
params = SCTConfiguration(perf_simple_query_extra_command=None, force_run_iotune=None, data_volume_disk_throughput=None, multi_...d_regions=['eu-west-1', 'eu-west-2', 'eu-west-3', 'us-west-2', 'us-east-1', 'us-east-2', 'eu-north-1', 'eu-central-1'])
gemini_schemas_dir = PosixPath('/home/ubuntu/scylla-cluster-tests/unit_tests/test_data/gemini_schemas')

    def test_gemini_write_only_with_oracle(request, docker_scylla, docker_scylla_oracle, params, gemini_schemas_dir):
        """A write-only run against an oracle cluster must succeed and produce non-zero write_ops.
    
        Write-only mode is used for CDC tests and write-only workloads where reads are not
        performed during the run. An oracle cluster is still present so that a subsequent
        read-mode run could validate consistency if needed.
        """
        thread = build_gemini_thread(
            params, docker_scylla, docker_scylla_oracle, gemini_schemas_dir=gemini_schemas_dir, mode="write", duration="30s"
        )
        request.addfinalizer(thread.kill)
        thread.run()
        results = thread.get_gemini_results()
        stats = thread.verify_gemini_results(results)
    
>       assert stats["status"] == "PASSED", f"Gemini write-only run failed: {stats}"
E       AssertionError: Gemini write-only run failed: {'results': [], 'errors': {}, 'status': 'FAILED'}
E       assert 'FAILED' == 'PASSED'
E         
E         - PASSED
E         + FAILED

unit_tests/integration/test_gemini_thread.py:385: AssertionError
```
</details>

#### `integration.test_ssh_none_auth.test_ssh_none_auth[libssh2]`
**Type:** ERROR
**Message:** `failed on setup with "subprocess.CalledProcessError: Command '['docker', 'build', '-t', 'ssh_server_none_auth_test:latest', PosixPath('/home/ubuntu/scylla-cluster-tests/unit_tests/test_data/auth_none_ssh_docker')]' returned non-zero exit status 137."`
<details><summary>Traceback</summary>

```
test_data_dir = PosixPath('/home/ubuntu/scylla-cluster-tests/unit_tests/test_data')

    @pytest.fixture(scope="module")
    def ssh_server_container(test_data_dir):
        """Start a Docker container with SSH server configured for 'none' authentication."""
        exposed_port = 22
        docker_dir = test_data_dir / "auth_none_ssh_docker"
        image_tag = "ssh_server_none_auth_test:latest"
    
>       subprocess.run(["docker", "build", "-t", image_tag, docker_dir], check=True)

unit_tests/integration/test_ssh_none_auth.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

input = None, capture_output = False, timeout = None, check = True
popenargs = (['docker', 'build', '-t', 'ssh_server_none_auth_test:latest', PosixPath('/home/ubuntu/scylla-cluster-tests/unit_tests/test_data/auth_none_ssh_docker')],)
kwargs = {}
process = <Popen: returncode: 137 args: ['docker', 'build', '-t', 'ssh_server_none_aut...>
stdout = None, stderr = None, retcode = 137

    def run(*popenargs,
            input=None, capture_output=False, timeout=None, check=False, **kwargs):
        """Run command with arguments and return a CompletedProcess instance.
    
        The returned instance will have attributes args, returncode, stdout and
        stderr. By default, stdout and stderr are not captured, and those attributes
        will be None. Pass stdout=PIPE and/or stderr=PIPE in order to capture them,
        or pass capture_output=True to capture both.
    
        If check is True and the exit code was non-zero, it raises a
        CalledProcessError. The CalledProcessError object will have the return code
        in the returncode attribute, and output & stderr attributes if those streams
        were captured.
    
        If timeout (seconds) is given and the process takes too long,
         a TimeoutExpired exception will be raised.
    
        There is an optional argument "input", allowing you to
        pass bytes or a string to the sub
```
</details>

[Full build log](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-15671/2/)

--- a/unit_tests/integration/test_external_backtrace_service.py
+++ b/unit_tests/integration/test_external_backtrace_service.py
@@ -16,8 +16,12 @@
 External services: HTTPS to backtrace.scylladb.com
 """
 
-import pytest
+from http import HTTPStatus
 
+import pytest
+import yaml
+
+from sdcm.utils.session import create_retry_session
 from unit_tests.lib.dummy_remote import DummyRemote
 from unit_tests.lib.fake_cluster import DummyNode
 
@@ -25,8 +29,10 @@ pytestmark = [
     pytest.mark.integration,
 ]
 
-# Known-good build from Scylla 2026.1.0~dev (SCYLLADB-403)
-KNOWN_BUILD_ID = "527bc25417c81c8106b33400a4372616c7c90894"
+LATEST_MASTER_BUILD_URL = (
+    "https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla/master/relocatable/latest/00-Build.txt"
+)
+# Addresses inside Scylla's text segment, so any build symbolizes them to something.
 SAMPLE_BACKTRACE_ADDRESSES = "0x4a3c9b7\n0x11ddd33\n0x11ddcec"
 
 
@@ -37,9 +43,33 @@ def node_fixture(tmp_path):
     return node
 
 
-def test_decode_via_external_service_returns_symbols(node):
+@pytest.fixture(name="known_build_id", scope="module")
+def known_build_id_fixture():
+    """Resolve a build the service can still symbolize.
+
+    A pinned build id rots: the service needs the build's unstripped package,
+    and unstable relocatables are garbage collected within a few months. Take
+    the current promoted master build instead - newer builds exist, but they
+    are not indexed by the service until they are promoted.
+    """
+    session = create_retry_session()
+
+    res = session.get(LATEST_MASTER_BUILD_URL, timeout=30)
+    res.raise_for_status()
+    build_id = yaml.safe_load(res.content)["scylla-x86_64-BuildID[sha1]"]
+
+    known = session.get("https://backtrace.scylladb.com/api/search/build_id", params={"build_id": build_id}, timeout=30)
+    # Only an unindexed build is a reason to skip. Anything else -- an outage, an
+    # auth error -- must fail loudly rather than quietly pass the suite.
+    if known.status_code == HTTPStatus.NOT_FOUND:
+        pytest.skip(f"backtrace.scylladb.com has not indexed the latest master build ({build_id}) yet")
+    known.raise_for_status()
+    return build_id
+
+
+def test_decode_via_external_service_returns_symbols(node, known_build_id):
     """Verify _decode_via_external_service returns decoded symbols for a known build."""
-    result = node._decode_via_external_service(KNOWN_BUILD_ID, SAMPLE_BACKTRACE_ADDRESSES)
+    result = node._decode_via_external_service(known_build_id, SAMPLE_BACKTRACE_ADDRESSES)
 
     assert result, "Decoded output should not be empty"
     assert "Backtrace" in result

